Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -393,7 +395,7 @@ import BraintreeDataCollector
 
         var urlComponents = URLComponents(url: payPalAppRedirectURL, resolvingAgainstBaseURL: true)
         urlComponents?.queryItems = [
-            URLQueryItem(name: "ba_token", value: baToken),
+            URLQueryItem(name: isVaultRequest ? "ba_token" : "token", value: token),
```

Comment:
On Android Sai and I noticed we don't append this at all there and instead get it from the URL. I am assuming `URLQueryItem` is smart enough to notice a delicate param and doesn't add it, but we may want to check the URL we get here and if the ba or ec token already exists can just append the source and time like on Android.

Reply:
ahh i see! i just tested the vault flow on prod and noticed that this param does indeed get appended to the url - i've addressed this here: 8e8275b5d7f414b3ecf4378cc6cc91bbe7dc7a06